src/StreamDeck/Transport/LibUSBHIDAPI.py

```diff
@@ -15,6 +16,39 @@
 from .Transport import Transport, TransportError
 
 
+def _find_lib_alt_location(homebrew_prefix: str | None, lib_name: str) -> str | None:
+    # If we've running with a Homebrew installation, and find_library() didn't find the library in
+    # any of the default search paths, we'll look in Homebrew instead as a fallback.
+
+    if homebrew_prefix:
+        library_path_homebrew = os.path.join(homebrew_prefix, 'lib', lib_name)
+        if os.path.exists(library_path_homebrew):
+            return library_path_homebrew
+
+    # We might also be in a Conda environment and have the library installed there.
+    conda_prefix = os.environ.get("CONDA_PREFIX")
+    if conda_prefix:
+        library_path_conda = os.path.join(conda_prefix, 'lib', lib_name)
+        if os.path.exists(library_path_conda):
+            return library_path_conda
+
+
+def _get_homebrew_path(self):
```

Comment:
I moved this function out of the class. I think there was no good reason to have it in the class.